Fixed earlier #3

ricariel opened the pull request:
If your change hook to prerouting you have to change chain name to something other than input.

henrythasler committed:
Add more customization options using shell variables

ricariel committed:
Added option for changing chain name

leshniak commented on nft-blacklist.sh:
```diff
@@ -4,7 +4,11 @@
 # eg: nft-blacklist.sh /etc/nft-blacklist/nft-blacklist.conf
 #
 
+CIDR_MERGER="cidr-merger" # can be "./cidr-merger-linux-amd64" or similar if you just donwloaded the binary
```
typo in the comment

drzraf commented:
Sounds like a sane PR. But I thing `CIDR_MERGER=iprange|cidr-merger` should be an option too so that Debian-based can keep using the "proven/official" IP-range-management  package.

ricariel committed:
setup actions

ricariel committed:
setup actions

ricariel committed:
setup actions

Commit:
chore(deps): pin https://code.forgejo.org/actions/forgejo-release action to 2eb5091

Commit:
fix(deps): update https://code.forgejo.org/actions/forgejo-release action ( v2.5.0 → v2.5.1 )

| datasource | package                 | from   | to     |
| ---------- | ----------------------- | ------ | ------ |
| gitea-tags | actions/forgejo-release | v2.5.0 | v2.5.1 |

ricariel committed:
renovate config

ricariel committed:
Bump version: 1.0.1 → 1.0.2
